src/PerfView/MainWindow.xaml.cs

```diff
@@ -1097,12 +1097,18 @@ public MainWindow()
                 }
                 if (StatusBar.IsWorking)
                 {
-                    var result = MessageBox.Show(this, "Rundown is still in progress. If you abort now the collected trace will be unusable. Are you sure you want to abort?", "Collecting data in progress", MessageBoxButton.YesNo);
-                    if (result == MessageBoxResult.No)
+                    if (App.CommandProcessor.RundownActive)
                     {
-                        e.Cancel = true;
-                        return;
+                        var result = MessageBox.Show(this,
+                            "Rundown is still in progress. If you abort now the collected trace will be unusable. Are you sure you want to abort?",
```

Comment:
This is more than just rundown, so I'd recommend going with something like "Closing PerfView while the trace is being processed will result in a trace that is unusable if copied off of this machine.  Would you still like to close PerfView?"  Will that be a reasonable prompt for the problem that you're trying to solve?

Reply:
Also recommend changing "RundownActive" to "StopInProgress".